projects/vdk-plugins/airflow-provider-vdk/vdk_provider/sensors/vdk.py

```diff
@@ -1,2 +1,29 @@
 # Copyright 2021 VMware, Inc.
 # SPDX-License-Identifier: Apache-2.0
+from typing import Any
+from typing import Dict
+
+from airflow.sensors.base import BaseSensorOperator
+from vdk_provider.hooks.vdk import VDKHook
+
+
+class VDKSensor(BaseSensorOperator):
+    def __init__(
```

Comment:
Can we pass workflow_id?

Reply:
The workflow_id can be extracted through the context object in the poke method.